Makefile

```diff
@@ -247,7 +247,7 @@ build-testnet-release:
 build-mainnet-release:
 	cargo build --locked --features  frequency --release
 
-.PHONY: test, e2e-tests, e2e-tests-serial, e2e-tests-only, e2e-tests-load, e2e-tests-load-only, e2e-tests-rococo, e2e-tests-rococo-local
+.PHONY: test, e2e-tests, e2e-tests-serial, e2e-tests-only, e2e-tests-load, e2e-tests-load-only, e2e-tests-rococo, e2e-tests-rococo-local, e2e-tests-testnet-paseo, e2e-tests-paseo-local
```

Comment:
Is `e2e-test-paseo-local` supposed to work out of the box in a local dev environment? Or what are the instructions;  it seems to expect Frequency to be running already...

Reply:
Paseo local is special. It does expect Frequency to already be running as setting up a full local relay chain is not easy.